mitmproxy/command.py

```diff
@@ -0,0 +1,91 @@
+import inspect
+import typing
+import shlex
+from mitmproxy.utils import typecheck
+from mitmproxy import exceptions
+from mitmproxy import flow
+
+
+def typename(t: type, ret: bool) -> str:
+    """
+        Translates a type to an explanatory string. Ifl ret is True, we're
+        looking at a return type, else we're looking at a parameter type.
+    """
+    if t in (str, int, bool):
+        return t.__name__
+    if t == typing.Sequence[flow.Flow]:
+        return "[flow]" if ret else "flowspec"
```

Comment:
Can we drop `ret` and just make this `[Flow]` for all cases? I think that's almost equally understandable for users and would provide more consistency across the board.
(we can then also move it to utils/typecheck)